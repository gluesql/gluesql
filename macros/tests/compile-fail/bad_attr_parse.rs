use gluesql_macros::FromGlueRow;

#[derive(FromGlueRow)]
struct S {
    #[glue]
    v: i32,
}
