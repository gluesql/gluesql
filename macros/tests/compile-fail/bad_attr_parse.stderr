error: expected attribute arguments in parentheses: #[glue(...)]
 --> tests/compile-fail/bad_attr_parse.rs:5:7
  |
5 |     #[glue]
  |       ^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compile-fail/bad_attr_parse.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/compile-fail/bad_attr_parse.rs`
